content/en/real_user_monitoring/generate_metrics/_index.md

```diff
@@ -0,0 +1,109 @@
+2. Create a search query that filters your RUM events using the RUM Explorer's [search syntax][7]. Events that contain a timestamp within the past hour and 15 minutes are included in the metric aggregation. 
+3. Choose a field to track from the dropdown menu next to **Count**. 
+
+    - Select `*` to generate a count of all RUM events that match your search query. 
+    - Optionally, enter an event attribute such as `@action.target` to aggregate a numeric value and create corresponding, aggregated metrics such as `count`, `min`, `max`, `sum`, and `avg`. 
+
+   If the RUM attribute facet is a measure, the metric value is the RUM attribute value.
+
+4. Select a path to group by from the dropdown menu next to **group by**. The metric tag name is the original attribute or tag name without the `@`. By default, metrics generated from RUM events do not contain tags unless they are explicitly added. You can use an attribute or tag dimension that exists in your RUM events such as `@error.source` or `env` to create metric tags. 
+   
+   <div class="alert alert-warning">Custom RUM metrics are considered as <a href="/metrics/custom_metrics/">custom metrics</a>. Datadog recommends avoiding grouping by unbounded or extremely high cardinality attributes such as timestamps, user IDs, request IDs, and session IDs. For more information, see <a href="/data_security/logs/">Log Management Data Security</a>.
+   </div>
+
+5. Add percentile aggregations for distribution metrics. You can generate P50, P75, P90, P95, and P99 percentiles. 
+
+   <div class="alert alert-warning">Percentile metrics are also considered <a href="/metrics/custom_metrics/">custom metrics</a>.
+
+6. Give your [metric][8] a name.
+7. Click **Create Metric**.
+
+Your custom RUM metric appears in the list below **Custom RUM Metrics**, and there may be a short delay for your metric to become available in [dashboards][9] and [monitors][10]. 
+
+Data points are not created for metrics with historical data. Data points for your custom RUM metric generate on a ten second interval, and metrics data is retained for 15 months. 
+
+### Update a custom RUM metric
+
+To update a metric, hover over a metric and click the **Edit** icon to the right hand corner.
+
+- Filter query: Change the set of matching RUM events that are aggregated into metrics.
+- Aggregation groups: Update tags or manage the cardinality of generated metrics.
+- Percentile selection: Click the **Calculate percentiles** toggle to remove or generate percentile metrics.
+
+Because you cannot rename a metric, Datadog recommends creating another metric.
+
+### Delete a custom RUM metric
+
+In order to stop the computing of data points from your metric (and billing), hover over a metric and click the **Delete** icon to the right hand corner. 
+
+## Use a custom RUM metric
```

Comment:
This is a matter of opinion, but "Usage" might be a simpler heading for this section.